rebasehelper/helpers/download_helper.py

```diff
@@ -152,6 +152,9 @@ def download_file(url, destination_path, blocksize=8192):
             raise DownloadError(r.reason)
 
         file_size = int(r.headers.get('content-length', -1))
+        encoding = r.headers.get('content-encoding', None)
+        if encoding and encoding != 'identity':
```

Comment:
You could squash this to a single line: `if r.headers.get('content-encoding', 'identity') != 'identity':`.

And I believe this deserves a comment, something like `use infinite progress bar in case of compressed content, since there is no way to determine the total uncompressed size in advance`.